aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -194,6 +194,10 @@ spec aptos_framework::account {
         aborts_if !from_bcs::deserializable<address>(new_auth_key_vector);
 
         aborts_if curr_auth_key != new_auth_key && table::spec_contains(address_map, new_auth_key);
+        ensures from_scheme == ED25519_SCHEME || from_scheme == MULTI_ED25519_SCHEME;
+
+        let post auth_key = global<Account>(addr).authentication_key;
+        ensures auth_key == new_auth_key_vector;
```

Comment:
Can you actually include `UpdateAuthKeyAndOriginatingAddressTableAbortsIf` because `update_auth_key_and_originating_address_table` is called in `rotate_authentication_key`?

Reply:
fixed